tmc-prototype/DishMaster/DishMaster/DishMaster.py

```diff
@@ -42,129 +34,127 @@ class DishMaster(SKAMaster):
+                print CONST.STR_ERR_MSG, except_occured
 
     def azimuth(self):
+        """ Calculates the azimuth angle difference. """
         self._pointing_state = 1
         self._azimuth_difference = self._desired_pointing[1] - self._achieved_pointing[1]
-        if self._azimuth_difference > 0.0:
-            self.increment_position([1,self._azimuth_difference])
-        elif self._azimuth_difference < 0.0:
-            self.decrement_position([1,abs(self._azimuth_difference)])
-        else:
-            pass
+        if self._azimuth_difference > 0.00:
+            self.increment_position([1, self._azimuth_difference])
+        elif self._azimuth_difference < 0.00:
+            self.decrement_position([1, abs(self._azimuth_difference)])
 
     def elevation(self):
+        """ Calculates the elevation angle difference. """
         self._pointing_state = 1
         self._elevation_difference = self._desired_pointing[2] - self._achieved_pointing[2]
-        if self._elevation_difference > 0.0:
-            self.increment_position([2,self._elevation_difference])
-        elif self._elevation_difference < 0.0:
-            self.decrement_position([2,abs(self._elevation_difference)])
-        else:
-            pass
-
-    def increment_position(self,argin):
-        temp = int(argin[1])
+        if self._elevation_difference > 0.00:
+            self.increment_position([2, self._elevation_difference])
+        elif self._elevation_difference < 0.00:
+            self.decrement_position([2, abs(self._elevation_difference)])
+
+    def increment_position(self, argin):
+        """
+        Increments the current pointing coordinates gradually to match the desired pointing coordinates.
+        :param argin: Difference between current and desired Azimuth/Elevation angle.
+        :return: None
+        """
+        input_increment = int(argin[1])
         time.sleep(2)
         if abs(self._azimuth_difference) > abs(self._elevation_difference):
-            temp1 = abs(self._azimuth_difference)
+            max_increment = abs(self._azimuth_difference)
         elif abs(self._azimuth_difference) < abs(self._elevation_difference):
-            temp1 = abs(self._elevation_difference)
-        else:
-            temp1 = temp
-            pass
-
-        if temp == temp1:
-            temp = temp + 1
+            max_increment = abs(self._elevation_difference)
         else:
-            pass
-
-        for position in range(0,temp):
-            self.set_status("Dish is pointing towards the desired coordinates.")
-            self.devlogmsg("Dish is pointing towards the desired coordinates.", 4)
+            max_increment = input_increment
+        if input_increment == max_increment:
+            input_increment = input_increment + 1
+        for position in range(0, input_increment):
+            self.set_status(CONST.STR_DISH_POINT_INPROG)
+            self.devlogmsg(CONST.STR_DISH_POINT_INPROG, int(tango.LogLevel.LOG_INFO))
```

Comment:
Make sure it is changed to _dev_logging_ everywhere.

Reply:
tmc-prototype is integrated with updated lmc base classes. This is there in the upcoming commit.